backend.native/tests/runtime/workers/leak_memory_with_worker_termination.kt

```diff
@@ -0,0 +1,9 @@
+import kotlin.native.concurrent.*
+import kotlinx.cinterop.*
+
+fun main() {
+    val worker = Worker.start()
+    Worker.current.park(1000 * 1000);
```

Comment:
There is a more reliable and efficient approach to ensure the worker is fully initialized: it is enough to schedule the task and wait for its result.